File: phase gate 1/increasingorder.py
numbers = [1,2,3]

sorted_numbers = [1,2,3]

flag = 0

numbers[0] = int(input("Enter your number : "))

numbers[1] = int(input("Enter your number : "))

numbers[2] = int(input("Enter your number : "))

largest = numbers[0]
smallest = numbers[0]

for counter in range(3):

	for count in range(3):

		if numbers[count]<smallest:

			smallest = numbers[count]


		if numbers[count]>largest:

			largest = numbers[count]
			flag = count

	
	sorted_numbers[2-counter] = largest
	largest = smallest
	numbers[flag]= smallest



print(sorted_numbers)

File: phase gate 1/IncreasingOrder.java
import java.util.Scanner;
public class IncreasingOrder{

    public static void main(String[] args){

	Scanner input = new Scanner(System.in);

	int [] numbers = new int [3];

	for(int count = 0; count <3;count++){

		System.out.println("Enter a number");

		numbers[count] = input.nextInt();

	}

	for(int count = 0 ;count< numbers.length; count++){
	
		for(int counter = 0; counter<numbers.length-1; counter++){

			if(numbers[counter]>numbers[counter+1]){

				numbers[counter+1]= numbers[counter] + numbers[counter+1];
				numbers[counter] = numbers[counter+1]- numbers[counter];
				numbers[counter+1] = numbers[counter+1] - numbers[counter];
		
			}
	
		}

		
	}


	for(int count = 0; count < numbers.length; count++){

		System.out.print(numbers[count]+" ");

	}




    }

}